<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupNewNetwork</class>
 <widget class="QWidget" name="SetupNewNetwork">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new network</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Neurons number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinNeurons">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Neurones entry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinEntry">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Threshol function</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="spinThreshold">
       <item>
        <property name="text">
         <string>Hyperbolic tangent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heavside</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sigmo√Øde</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nameText">
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>SetupNewNetwork</receiver>
   <slot>getNewNetwork()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SetupNewNetwork</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getNewNetwork()</slot>
 </slots>
</ui>
